marpa/lib/standard.pm

```diff
@@ -21,5 +27,31 @@ sub import ( $class ) {
     return;
 }
 
+sub strip_terminators ($contentref) {
+    ${$contentref} =~ s/^__DATA__\n.*//xms;
+    ${$contentref} =~ s/^__END__\n.*//xms;
+}
+
+sub strip_pods ($contentref) {
```

Comment:
Blunt tool, but if it does the job, I'm all for it... :-)

Reply:
There are POD parsers and one to strip POD from Perl code, but I figured, it's simple enough to write.